giskard/llm/evaluators/base.py

```diff
@@ -90,18 +93,18 @@ def evaluate(self, model: BaseModel, dataset: Dataset):
             try:
                 out = self.llm_client.complete(
                     [{"role": "system", "content": prompt}],
-                    functions=funcs,
-                    function_call={"name": "evaluate_model"},
+                    tools=funcs,
+                    tool_choice={"type": "function", "function": {"name": "evaluate_model"}},
                     temperature=self.llm_temperature,
                     caller_id=self.__class__.__name__,
                 )
-                if out.function_call is None or "passed_test" not in out.function_call.args:
+                if len(out.tool_calls) != 1 or "passed_test" not in out.tool_calls[0].args:
```

Comment:
In this specific case, we do not expect, that we can make multiple tool calls, right?

Reply:
Yes since we are evaluating a single case we expect it to be evaluated once